package.json

```diff
@@ -10,4 +10,4 @@
     "build-client": "bun run --cwd ./client build"
   },
   "dependencies": {}
-}
\ No newline at end of file
+}
```

Comment:
unnecessary newline?

Reply:
What formatter do you use? Mine's adding it

Reply:
kk, if its a formatting thing then we keep